README.md

```diff
@@ -43,7 +43,30 @@ Both the source and destination connectors share these configuration parameters:
 (*) Valid time units are "ns", "us" (or "µs"), "ms", "s", "m", "h".
 
 
-Example of a `pipeline.yml` file using `file to activemq classic` and `activemq classic to file` pipelines: 
+### Source configuration
+
+| name | description | required | default value |
+| ---- | ----------- | -------- | ------------- |
+| `clientID`                   | Specifies the JMS clientID which is used in combination with the activemq.subscriptionName to denote a durable subscriber. | No       |               |
+| `dispatchAsync`              | Specifies whether messages should be dispatched synchronously or asynchronously from the producer thread for non-durable topics in the broker. | No       | |
```

Comment:
I'd consider renaming the config parameter to `activemq.dispatchAsync` so it matches exactly the header it's setting. Same for the others that start with `activemq.*`. And for each one I'd call out what header will be set, so it's super explicit.